_tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

```diff
@@ -54,12 +54,44 @@ In addition to the mandatory static settings, you can configure the following dy
 
 Setting | Default | Description
 :--- | :--- | :---
-`cluster.remote_store.state.index_metadata.upload_timeout` | 20s | The amount of time to wait for index metadata upload to complete. Note that index metadata for separate indexes is uploaded in parallel.
-`cluster.remote_store.state.global_metadata.upload_timeout` | 20s | The amount of time to wait for global metadata upload to complete. Global metadata contains globally applicable metadata, such as templates, cluster settings, data stream metadata, and repository metadata.
-`cluster.remote_store.state.metadata_manifest.upload_timeout` | 20s | The amount of time to wait for the manifest file upload to complete. The manifest file contains the details of each of the files uploaded for a single cluster state, both index metadata files and global metadata files. 
+`cluster.remote_store.state.index_metadata.upload_timeout` | 20s | Deprecated. Use `cluster.remote_store.state.global_metadata.upload_timeout` instead.
+`cluster.remote_store.state.global_metadata.upload_timeout` | 20s | The amount of time to wait for cluster state upload to complete.
+`cluster.remote_store.state.metadata_manifest.upload_timeout` | 20s | The amount of time to wait for the manifest file upload to complete. The manifest file contains the details of each of the files uploaded for a single cluster state, both index metadata files and global metadata files.
+`cluster.remote_store.state.cleanup_interval` | 300s | The interval for remote state clean-up async task to run. This task deletes the old remote state files. 
 
 
 ## Limitations
 
 The remote cluster state functionality has the following limitations:
 - Unsafe bootstrap scripts cannot be run when the remote cluster state is enabled. When a majority of cluster-manager nodes are lost and the cluster goes down, the user needs to replace any remaining cluster manager nodes and reseed the nodes in order to bootstrap a new cluster.
+
+## Remote Cluster State Publication
+The cluster state published to remote-backed storage can be used for publication. Currently, the active cluster manager
```

Comment:
The cluster manager node processes the updates to cluster state. It then, publishes the updated cluster state over the local transport layer to all the follower nodes. With remote cluster state enabled, cluster state is backed to remote store with every state update. The follower nodes can fetch the state from remote store directly and reducing the overhead on the cluster manager node for publication. This can be done by enabling the experimental remote publication feature. 
Enable the feature flag for `remote_store.publication` feature by following the [experiment feature flag documentation]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/). This doesn't change the publication flow and follower nodes will not send acknowledgement back to cluster manager until they download the updated cluster state from remote store and proceed as expected in current flow. Please note enabling remote cluster state is mandatory for remote publication to work. 
Also, RoutingTable which contains the shard allocation details for each index in the cluster state requires setting up the remote blobstore repository. It can be configured as below: